app/assets/javascripts/application.js

```diff
@@ -13,8 +13,12 @@
 //= require jquery
 //= require jquery_ujs
 //= require jquery.ui.all
-//= require_tree .
+//= require ./do
+//= require ./jquery_extensions
+//= require ./ui
+//= require ./ox-account
```

Comment:
A tiny bit of it does.  It needs to:
 * observe that it's loaded inside an iframe and then apply the `iframe` css class to `body`
 * Fire a page load notification to the parent of the iframe so it can possibly close the iframe.
 * installs a window.resize listener so the iframe can avoid scroll bars.

I'm planning to de-couple that so it can only load what's needed.  Right now it's tucked in the `initWebsite` method of `host.coffee`